content/git/concepts/reset/reset.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: 'Reset'
+Description: 'In Git, reset is used to set the HEAD pointer back to a previous commit and optionally undo staged changes and the working tree.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'Version Control'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+---
+
+In Git, the `reset` command is used to change the state of the Git repository or undo commits.
+
+## Syntax
+
+In the terminal, the "git reset" command can be used by typing the following:
+
+```pseudo
+git reset <mode-option> <commit-reference-number>
+```
+
+### Mode Options
+
+The `<mode-options>` refer to how far `reset` will go in when rolling back changes, in the following contexts:
+
+- Where the `HEAD` is pointing towards (usually done with just `git reset`).
+- Whether the staging area, or Index, reflects the commit the `HEAD` is now pointing towards.
+- Whether the working tree is rolled back to reflect the changes reset in the `HEAD` and staging area.
+
+More specifically, these options include:
+
+1. --soft: This undoes the commit but still shows the file as staged and permits only new commits.
+2. --mixed: This undoes the commit and unstages the file and the file can be edited and committed normally.
+3. --hard: This removes the new commit including all changes and uncommitted files.
+
+The commit can either be the commit hash generated by Git, or in reference to the HEAD.
```

Comment:
Can we include a brief snippet below this sentence that supports this idea?

```git
git reset --hard 7afd58d97c65c1f6df703e4ca5290747

git reset --hard HEAD~
```

Reply:
> Can we include a brief snippet below this sentence that supports this idea?
> 
> ```
> git reset --hard 7afd58d97c65c1f6df703e4ca5290747
> 
> git reset --hard HEAD~
> ```

Do you want it like a syntax or a redo of the entire process ?

Reply:
Actually, we can probably omit this line because I suggested an edit up top in the "## Syntax" section that uses almost the same wording.

Reply:
I'm going to go ahead and remove this line since it's redundant 👍

Reply:
> Actually, we can probably omit this line because I suggested an edit up top in the "## Syntax" section that uses almost the same wording.
> I'm going to go ahead and remove this line since it's redundant 👍

So do you want me to delete line 63-66? As for the edit request at syntax, what do I add as it does mention the mode and the commit commit reference.